spec/models/filter/getMethods.spec.ts

```diff
@@ -0,0 +1,62 @@
+import {FilterModel, FilterModelProperty} from "../../../src/models/filter/filter-model";
+import {FilterConfig, FilterConfigItem} from "../../../src/models/filter/filter-config-item";
+import {TestItem} from "./test-item";
+
+describe('Filter model - get methods:', () => {
```

Comment:
Please note that there is no need to test simple getter if they don't contain any conditional logic.